augur/utils.py

```diff
@@ -432,6 +434,21 @@ def _read_genbank(reference, feature_names):
             if fname and (feature_names is None or fname in feature_names):
                 features[fname] = feat
 
+        if feat.type=='gene':
```

Comment:
To reduce the chances of this introducing unexpected changes to current usage, I'd suggest an approach where we only use 'gene' features if the corresponding 'CDS' feature doesn't exist. There's a few different ways to implement this, but the simplest may be to use an extra loop through `gb.features`.

Reply:
Continuing this discussion based on https://github.com/nextstrain/rsv/pull/55#issuecomment-2012999889:

Theoretically CDSs are what we want, but [we have GenBank files which use "gene" instead](https://github.com/nextstrain/measles/blob/9377ef6623cf3b5ea779ef3a3244f98b76ddc426/phylogenetic/defaults/measles_reference.gb#L71-L86). I can't think of a situation  where we want to extract _both_ from the same genbank reference¹. One option is to look for 'gene' only if no CDSs are found. Another is to expose this as a command line argument to augur.

¹ I'm sure there are exceptions to this, in which case I think the onus is on the user to edit the GenBank file upstream of Augur.

Reply:
Is this something where we need to poll users to vote for "CDS" or "gene"? Or it sounds like we're pretty sure it's "CDS"?

> Another is to expose this as a command line argument to augur.

Ah, I could get behind having a command line argument